### PR TITLE
Support building on windows

### PR DESCRIPTION
**Pull request by turtleDeng**

### CODE REVIEW

File: src/crc32cer.erl [hunk 1]
```diff
@@ -24,24 +24,15 @@ nif(_Acc, _IoData) ->
...
-    ]).
-
-get_nif_bin_dir([]) -> erlang:error(crc32cer_nif_not_found);
-get_nif_bin_dir([false | Rest]) -> get_nif_bin_dir(Rest);
-get_nif_bin_dir([Dir | Rest]) ->
-  case filelib:wildcard(filename:join([Dir, "crc32cer.*"])) of
-    [] -> get_nif_bin_dir(Rest);
-    [_ | _] -> Dir
-  end.
+  filename:join(case code:priv_dir(?MODULE) of
```
Comment (zmstone) [inline src/crc32cer.erl:27]: not sure why we need this
@gilbertwong96

Comment (zmstone) [reply src/crc32cer.erl:27]: or why the original code does not work?

Comment (gilbertwong96) [reply src/crc32cer.erl:27]: Sorry, I totally forget about the meaning of these commits. It’s should be something wrong happened on windows and I hacked them with these codes.

Comment (zmstone) [reply src/crc32cer.erl:27]: I would prefer to have this reverted, after merge
could you help to verify if it works in windows ? @gilbertwong96

**Pull request merged**